### ISSUE TITLE
oAuth 1.0 Authorization HTTP header leading comma

### ISSUE THREAD

**Issue by EpocDotFr** [a85/POSTMan-Chrome-Extension#2312]
1. Postman Version: 4.7.0
2. App (Chrome app or Mac app): Both
3. OS details: Windows 10
4. Is the Interceptor on and enabled in the app: No
5. Did you encounter this recently, or has this bug always been there: Encountered recently
6. Expected behaviour: oAuth 1.0 HTTP header is working

Using the latest Windows app, try to use oAuth 1.0 with the following parameters: Consumer key, Consumer secret while having only the "Add params to header" checked. You'll see a comma prepended in the `Authorization` HTTP header value, which is invalid. Example:

```
Authorization:"OAuth ,oauth_consumer_key="55c2XXXXXXXX",oauth_signature_method="HMAC-SHA1",oauth_timestamp="1473175779",oauth_nonce="bHSXXXXXXXX",oauth_version="1.0",oauth_signature="XXXXQeePWutLXXXX%2BUbXXXX6cBXXXX""
                     ^
```

Please note that this issue is a regression as oAuth 1.0 was working fine before.

**Comment by ZeWaren** [comment]
Same thing here.
The good ol' Chrome app doesn't have this issue, but the windows packaged one does.

**Comment by EpocDotFr** [comment]
FYI this bug is blocking and prevent using the oAuth 1.0 feature.

**Comment by EpocDotFr** [comment]
I just installed Chrome with the Postman extension. Exact same issue. @ZeWaren surely hasn't the latest version installed.

So this bug is now EXTREMELY blocking, in addition to be a regression.

**Comment by sdnts** [comment]
@EpocDotFr @ZeWaren We're looking into this, aiming to push out a fix ASAP. Will let you know here as soon as it is out.

**Comment by EpocDotFr** [comment]
I found a workaround that will only work if your oAuth 1.0 server implements this behavior.

Just uncheck the **Add params to header** option, and it will works. This will no more create the `Authorization` header but instead put all oAuth parameters in the URL query string.

**Comment by sdnts** [comment]
@EpocDotFr We released 4.7.2 yesterday with a fix for this. could you confirm?

**Comment by EpocDotFr** [comment]
@madebysid All good for me :ok_hand:
